core/server/worker/src/main/java/alluxio/worker/page/PagedBlockMetaStore.java

```diff
@@ -404,6 +404,26 @@ public PagedBlockStoreMeta getStoreMetaFull() {
         usedBytesOnDirs.build(), blockOnDirs.build());
   }
 
+  /**
+   * Report all block locations in a pageStoreDirs to master using BlockHeartbeat.
+   */
+  @GuardedBy("getLock().readLock()")
+  public void reportBlocks() {
```

Comment:
The actual reporting does not happen inside this method, so may be rename this to reflect what it actually does?

```suggestion
  /**
   * Callback that gets called when the underlying page store finishes async
   * restoration.
   * This method notifies all registered block store event listeners
   * about the newly added blocks. In particular, these blocks will be reported
   * as added blocks to master via heartbeat.
   */
  @GuardedBy("getLock().readLock()")
  public void onCacheRestorationSuccess() {
```